core/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestNodesStatsAction.java

```diff
@@ -114,8 +115,16 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         return channel -> client.admin().cluster().nodesStats(nodesStatsRequest, new NodesResponseRestListener<>(channel));
     }
 
+    private final Set<String> RESPONSE_PARAMS = Collections.singleton("level");
+
+    @Override
+    protected Set<String> responseParams() {
+        return RESPONSE_PARAMS;
+    }
+
```

Comment:
I see. I was wondering if there was a way to catch that we forgot about it. We test that whitelisted settings work I think, so now that it is whitelisted it works for sure, but what if we un-whitelist it by mistake? :) do you see what I mean? That said I am not sure, maybe we could add a small rest tests to the existing ones that leverage this param?